packages/a11y/src/validateProps.tsx

```diff
@@ -11,3 +11,17 @@ export function validateAriaLabelProps(
     );
   }
 }
+
+// Validate that either label or aria-labelledby are present within props.
+export function validateLabelProps(
+  props: Record<string, any>,
+  componentName?: string,
```

Comment:
We should probably make componentName required considering the number of times this being optional has been a pain.

Reply:
Good call